README.md

```diff
@@ -11,7 +11,7 @@ If you want to contribute or need help getting started, join us on Slack by fill
 To develop this extension the following environment is used
```

Comment:
Ohh yeah you are right I think its `npm run build` ?
I think it would be useful if we could document our npm script here as well with short descriptions, at least the main ones